ABDefinitionCore.js

```diff
@@ -42,6 +42,15 @@ module.exports = class ABDefinitionCore {
       }
       this.type = attributes.type || attributes?.json?.type || "";
       this.json = attributes.json || null;
+
+      this.version = attributes.version || attributes.json?.version || "1.0.0";
```

Comment:
I don't think we need to store this information at the `ABDefinition` level.  

Currently versioning is for `ABApplication`s and you already have it being pulled out and defined there.   Adding it in here will attempt to pull out version info for EVERYTHING in our AppBuilder definitions: ABObject, ABFields, ABQueries, ABDatacollections, tasks, etc...  

So let's just leave it at the `ABApplication` level.